tools/Config.mk

```diff
@@ -329,8 +349,7 @@ endef
 #   CONFIG_WINDOWS_NATIVE - Defined for a Windows native build
 
 define ARCHIVE_ADD
-	@echo "AR (add): ${shell basename $(1)} $(2)"
```

Comment:
can't we still have a print here? Otherwise nothing will be printed

Reply:
No the argument list will be too long

Reply:
But maybe the print can be done in batches as well

Reply:
I'm not really sure how that would work, if I wrap the echo with max args then it will split differently.

Reply:
I was thinking that it could show as "AR (add): <lib> <up to 100 objs here>". But if you think it is too complex, maybe, just leave the "AR (add): <lib>". Otherwise we can't tell when this being done.

Reply:
Ok I reworked the function a bit so it can take a functions as an argument.  This means we can retain something like the original functionality even with it split. Here is an output, we see both the split and not split versions:
```
make[3]: Entering directory '/home/bashton/nuttx/wrk/apps/netutils/ftpc'
AR (add): libapps.a ftpc_connect.home.bashton.nuttx.wrk.apps.netutils.ftpc.o ftpc_disconnect.home.bashton.nuttx.wrk.apps.netutils.ftpc.o ftpc_cdup.home.bashton.nuttx.wrk.apps.netutils.ftpc.o ftpc_chdir.home.bashton.nuttx.wrk.apps.netutils.ftpc.o ftpc_chmod.home.bashton.nuttx.wrk.apps.netutils.ftpc.o ftpc_filesize.home.bashton.nuttx.wrk.apps.netutils.ftpc.o ftpc_filetime.home.bashton.nuttx.wrk.apps.netutils.ftpc.o ftpc_help.home.bashton.nuttx.wrk.apps.netutils.ftpc.o ftpc_idle.home.bashton.nuttx.wrk.apps.netutils.ftpc.o ftpc_listdir.home.bashton.nuttx.wrk.apps.netutils.ftpc.o ftpc_login.home.bashton.nuttx.wrk.apps.netutils.ftpc.o ftpc_mkdir.home.bashton.nuttx.wrk.apps.netutils.ftpc.o ftpc_noop.home.bashton.nuttx.wrk.apps.netutils.ftpc.o ftpc_rpwd.home.bashton.nuttx.wrk.apps.netutils.ftpc.o ftpc_quit.home.bashton.nuttx.wrk.apps.netutils.ftpc.o ftpc_rename.home.bashton.nuttx.wrk.apps.netutils.ftpc.o ftpc_rmdir.home.bashton.nuttx.wrk.apps.netutils.ftpc.o ftpc_unlink.home.bashton.nuttx.wrk.apps.netutils.ftpc.o ftpc_cmd.home.bashton.nuttx.wrk.apps.netutils.ftpc.o ftpc_getfile.home.bashton.nuttx.wrk.apps.netutils.ftpc.o ftpc_putfile.home.bashton.nuttx.wrk.apps.netutils.ftpc.o ftpc_transfer.home.bashton.nuttx.wrk.apps.netutils.ftpc.o ftpc_response.home.bashton.nuttx.wrk.apps.netutils.ftpc.o ftpc_getreply.home.bashton.nuttx.wrk.apps.netutils.ftpc.o ftpc_utils.home.bashton.nuttx.wrk.apps.netutils.ftpc.o ftpc_socket.home.bashton.nuttx.wrk.apps.netutils.ftpc.o
make[3]: Leaving directory '/home/bashton/nuttx/wrk/apps/netutils/ftpc'
make[3]: Entering directory '/home/bashton/nuttx/wrk/apps/netutils/netinit'
AR (add): libapps.a netinit.home.bashton.nuttx.wrk.apps.netutils.netinit.o
make[3]: Leaving directory '/home/bashton/nuttx/wrk/apps/netutils/netinit'
make[3]: Entering directory '/home/bashton/nuttx/wrk/apps/netutils/netlib'
AR (add): libapps.a netlib_ipv4addrconv.home.bashton.nuttx.wrk.apps.netutils.netlib.o netlib_ethaddrconv.home.bashton.nuttx.wrk.apps.netutils.netlib.o netlib_parsehttpurl.home.bashton.nuttx.wrk.apps.netutils.netlib.o netlib_setifstatus.home.bashton.nuttx.wrk.apps.netutils.netlib.o netlib_getifstatus.home.bashton.nuttx.wrk.apps.netutils.netlib.o netlib_parseurl.home.bashton.nuttx.wrk.apps.netutils.netlib.o netlib_setipv4addr.home.bashton.nuttx.wrk.apps.netutils.netlib.o netlib_getipv4addr.home.bashton.nuttx.wrk.apps.netutils.netlib.o netlib_setdripv4addr.home.bashton.nuttx.wrk.apps.netutils.netlib.o netlib_setipv4netmask.home.bashton.nuttx.wrk.apps.netutils.netlib.o netlib_getdripv4addr.home.bashton.nuttx.wrk.apps.netutils.netlib.o netlib_getipv4netmask.home.bashton.nuttx.wrk.apps.netutils.netlib.o netlib_ipv4adaptor.home.bashton.nuttx.wrk.apps.netutils.netlib.o netlib_getarp.home.bashton.nuttx.wrk.apps.netutils.netlib.o netlib_setarp.home.bashton.nuttx.wrk.apps.netutils.netlib.o netlib_delarp.home.bashton.nuttx.wrk.apps.netutils.netlib.o netlib_getarptab.home.bashton.nuttx.wrk.apps.netutils.netlib.o netlib_setipv4dnsaddr.home.bashton.nuttx.wrk.apps.netutils.netlib.o netlib_setipv6addr.home.bashton.nuttx.wrk.apps.netutils.netlib.o netlib_getipv6addr.home.bashton.nuttx.wrk.apps.netutils.netlib.o netlib_setdripv6addr.home.bashton.nuttx.wrk.apps.netutils.netlib.o netlib_setipv6netmask.home.bashton.nuttx.wrk.apps.netutils.netlib.o netlib_prefix2ipv6netmask.home.bashton.nuttx.wrk.apps.netutils.netlib.o netlib_ipv6netmask2prefix.home.bashton.nuttx.wrk.apps.netutils.netlib.o netlib_ipv6adaptor.home.bashton.nuttx.wrk.apps.netutils.netlib.o netlib_autoconfig.home.bashton.nuttx.wrk.apps.netutils.netlib.o netlib_getnbtab.home.bashton.nuttx.wrk.apps.netutils.netlib.o netlib_getdevs.home.bashton.nuttx.wrk.apps.netutils.netlib.o netlib_server.home.bashton.nuttx.wrk.apps.netutils.netlib.o netlib_listenon.home.bashton.nuttx.wrk.apps.netutils.netlib.o netlib_setmacaddr.home.bashton.nuttx.wrk.apps.netutils.netlib.o netlib_getmacaddr.home.bashton.nuttx.wrk.apps.netutils.netlib.o netlib_ipmsfilter.home.bashton.nuttx.wrk.apps.netutils.netlib.o
make[3]: Leaving directory '/home/bashton/nuttx/wrk/apps/netutils/netlib'
make[3]: Entering directory '/home/bashton/nuttx/wrk/apps/netutils/ping'
AR (add): libapps.a icmp_ping.home.bashton.nuttx.wrk.apps.netutils.ping.o icmpv6_ping.home.bashton.nuttx.wrk.apps.netutils.ping.o
make[3]: Leaving directory '/home/bashton/nuttx/wrk/apps/netutils/ping'
make[3]: Entering directory '/home/bashton/nuttx/wrk/apps/netutils/telnetc'
AR (add): libapps.a telnetc.home.bashton.nuttx.wrk.apps.netutils.telnetc.o
make[3]: Leaving directory '/home/bashton/nuttx/wrk/apps/netutils/telnetc'
make[3]: Entering directory '/home/bashton/nuttx/wrk/apps/netutils/telnetd'
AR (add): libapps.a telnetd_daemon.home.bashton.nuttx.wrk.apps.netutils.telnetd.o
make[3]: Leaving directory '/home/bashton/nuttx/wrk/apps/netutils/telnetd'
make[3]: Entering directory '/home/bashton/nuttx/wrk/apps/netutils/tftpc'
AR (add): libapps.a tftpc_get.home.bashton.nuttx.wrk.apps.netutils.tftpc.o tftpc_put.home.bashton.nuttx.wrk.apps.netutils.tftpc.o tftpc_packets.home.bashton.nuttx.wrk.apps.netutils.tftpc.o
make[3]: Leaving directory '/home/bashton/nuttx/wrk/apps/netutils/tftpc'
make[3]: Entering directory '/home/bashton/nuttx/wrk/apps/netutils/webclient'
AR (add): libapps.a webclient.home.bashton.nuttx.wrk.apps.netutils.webclient.o
make[3]: Leaving directory '/home/bashton/nuttx/wrk/apps/netutils/webclient'
make[3]: Entering directory '/home/bashton/nuttx/wrk/apps/nshlib'
AR (add): libapps.a nsh_init.home.bashton.nuttx.wrk.apps.nshlib.o nsh_parse.home.bashton.nuttx.wrk.apps.nshlib.o nsh_console.home.bashton.nuttx.wrk.apps.nshlib.o nsh_script.home.bashton.nuttx.wrk.apps.nshlib.o nsh_system.home.bashton.nuttx.wrk.apps.nshlib.o nsh_command.home.bashton.nuttx.wrk.apps.nshlib.o nsh_fscmds.home.bashton.nuttx.wrk.apps.nshlib.o nsh_ddcmd.home.bashton.nuttx.wrk.apps.nshlib.o nsh_proccmds.home.bashton.nuttx.wrk.apps.nshlib.o nsh_mmcmds.home.bashton.nuttx.wrk.apps.nshlib.o nsh_timcmds.home.bashton.nuttx.wrk.apps.nshlib.o nsh_envcmds.home.bashton.nuttx.wrk.apps.nshlib.o nsh_syscmds.home.bashton.nuttx.wrk.apps.nshlib.o nsh_dbgcmds.home.bashton.nuttx.wrk.apps.nshlib.o nsh_session.home.bashton.nuttx.wrk.apps.nshlib.o nsh_fsutils.home.bashton.nuttx.wrk.apps.nshlib.o nsh_builtin.home.bashton.nuttx.wrk.apps.nshlib.o nsh_fileapps.home.bashton.nuttx.wrk.apps.nshlib.o nsh_romfsetc.home.bashton.nuttx.wrk.apps.nshlib.o nsh_netcmds.home.bashton.nuttx.wrk.apps.nshlib.o nsh_mntcmds.home.bashton.nuttx.wrk.apps.nshlib.o nsh_consolemain.home.bashton.nuttx.wrk.apps.nshlib.o nsh_telnetd.home.bashton.nuttx.wrk.apps.nshlib.o nsh_test.home.bashton.nuttx.wrk.apps.nshlib.o
make[3]: Leaving directory '/home/bashton/nuttx/wrk/apps/nshlib'
make[3]: Entering directory '/home/bashton/nuttx/wrk/apps/platform'
AR (add): libapps.a dummy.home.bashton.nuttx.wrk.apps.platform.o
make[3]: Leaving directory '/home/bashton/nuttx/wrk/apps/platform'
make[3]: Entering directory '/home/bashton/nuttx/wrk/apps/system/cle'
AR (add): libapps.a cle.home.bashton.nuttx.wrk.apps.system.cle.o
make[3]: Leaving directory '/home/bashton/nuttx/wrk/apps/system/cle'
make[3]: Entering directory '/home/bashton/nuttx/wrk/apps/system/nsh'
AR (add): libapps.a nsh_main.home.bashton.nuttx.wrk.apps.system.nsh.o sh_main.home.bashton.nuttx.wrk.apps.system.nsh.o
make[3]: Leaving directory '/home/bashton/nuttx/wrk/apps/system/nsh'
make[3]: Entering directory '/home/bashton/nuttx/wrk/apps/system/ping6'
AR (add): libapps.a ping6.home.bashton.nuttx.wrk.apps.system.ping6.o
make[3]: Leaving directory '/home/bashton/nuttx/wrk/apps/system/ping6'
make[3]: Entering directory '/home/bashton/nuttx/wrk/apps/system/ping'
AR (add): libapps.a ping.home.bashton.nuttx.wrk.apps.system.ping.o
make[3]: Leaving directory '/home/bashton/nuttx/wrk/apps/system/ping'
make[3]: Entering directory '/home/bashton/nuttx/wrk/apps/system/popen'
AR (add): libapps.a popen.home.bashton.nuttx.wrk.apps.system.popen.o
make[3]: Leaving directory '/home/bashton/nuttx/wrk/apps/system/popen'
make[3]: Entering directory '/home/bashton/nuttx/wrk/apps/system/readline'
AR (add): libapps.a readline.home.bashton.nuttx.wrk.apps.system.readline.o readline_common.home.bashton.nuttx.wrk.apps.system.readline.o
make[3]: Leaving directory '/home/bashton/nuttx/wrk/apps/system/readline'
make[3]: Entering directory '/home/bashton/nuttx/wrk/apps/system/system'
AR (add): libapps.a system.home.bashton.nuttx.wrk.apps.system.system.o
make[3]: Leaving directory '/home/bashton/nuttx/wrk/apps/system/system'
make[3]: Entering directory '/home/bashton/nuttx/wrk/apps/system/telnet'
AR (add): libapps.a telnet_client.home.bashton.nuttx.wrk.apps.system.telnet.o
make[3]: Leaving directory '/home/bashton/nuttx/wrk/apps/system/telnet'
make[3]: Entering directory '/home/bashton/nuttx/wrk/apps/testing/ltp'
AR (add): libapps.a ltp/testcases/open_posix_testsuite/conformance/definitions/sys/mman_h/5-2-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/sys/mman_h/16-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/sys/mman_h/15-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/sys/mman_h/3-2-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/sys/mman_h/24-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/sys/mman_h/3-3-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/sys/mman_h/8-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/sys/mman_h/13-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/sys/mman_h/2-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/sys/mman_h/14-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/sys/mman_h/9-2-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/sys/mman_h/4-2-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/sys/mman_h/20-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/sys/mman_h/4-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/sys/mman_h/2-2-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/sys/mman_h/5-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/sys/mman_h/7-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/sys/mman_h/3-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/sys/mman_h/23-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/sys/mman_h/2-4-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/sys/mman_h/1-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/sys/mman_h/7-2-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/sys/mman_h/22-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/sys/mman_h/6-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/sys/mman_h/8-3-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/sys/mman_h/8-2-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/sys/mman_h/10-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/sys/mman_h/2-3-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/sys/mman_h/12-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/sys/mman_h/18-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/sys/mman_h/4-3-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/sys/mman_h/9-3-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/sys/mman_h/9-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/sys/shm_h/8-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/sys/shm_h/2-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/sys/shm_h/4-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/sys/shm_h/2-2-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/sys/shm_h/5-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/sys/shm_h/7-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/sys/shm_h/3-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/sys/shm_h/1-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/sys/shm_h/7-2-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/sys/shm_h/12-3-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/sys/shm_h/6-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/sys/shm_h/12-2-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/sys/shm_h/10-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/sys/shm_h/12-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/sys/shm_h/9-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/mqueue_h/8-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/mqueue_h/2-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/mqueue_h/4-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/mqueue_h/5-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/mqueue_h/7-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/mqueue_h/3-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/mqueue_h/6-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/mqueue_h/10-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/mqueue_h/9-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/signal_h/46-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/signal_h/15-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/signal_h/34-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/signal_h/28-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/signal_h/24-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/signal_h/41-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/signal_h/22-40-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/signal_h/47-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/signal_h/44-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/signal_h/2-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/signal_h/32-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/signal_h/22-25-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/signal_h/22-37-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/signal_h/22-39-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/signal_h/4-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/signal_h/48-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/signal_h/2-2-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/signal_h/29-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/signal_h/5-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/signal_h/22-29-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/signal_h/3-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/signal_h/2-4-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/signal_h/50-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/signal_h/1-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/signal_h/22-24-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/signal_h/30-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/signal_h/36-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/signal_h/40-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/signal_h/39-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/signal_h/38-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/signal_h/33-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/signal_h/45-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/signal_h/22-27-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/signal_h/22-38-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/signal_h/2-3-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/signal_h/22-26-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/signal_h/43-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/signal_h/49-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/signal_h/22-28-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/signal_h/42-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/signal_h/22-36-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/unistd_h/2-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/unistd_h/1-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o
AR (add): libapps.a ltp/testcases/open_posix_testsuite/conformance/definitions/sched_h/16-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/sched_h/15-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/sched_h/8-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/sched_h/13-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/sched_h/2-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/sched_h/14-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/sched_h/4-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/sched_h/3-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/sched_h/1-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/sched_h/8-3-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/sched_h/8-2-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/sched_h/12-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/sched_h/8-4-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/sched_h/18-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/errno_h/2-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/errno_h/3-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/errno_h/1-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/time_h/16-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/time_h/15-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/time_h/31-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/time_h/34-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/time_h/3-2-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/time_h/28-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/time_h/24-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/time_h/26-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/time_h/8-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/time_h/13-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/time_h/2-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/time_h/32-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/time_h/14-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/time_h/20-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/time_h/4-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/time_h/29-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/time_h/5-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/time_h/7-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/time_h/3-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/time_h/23-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/time_h/6-2-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/time_h/1-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/time_h/30-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/time_h/7-2-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/time_h/22-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/time_h/33-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/time_h/6-3-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/time_h/6-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/time_h/10-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/time_h/12-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/time_h/18-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/time_h/35-3-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/time_h/9-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/pthread_h/16-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/pthread_h/3-11-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/pthread_h/15-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/pthread_h/3-12-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/pthread_h/3-2-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/pthread_h/3-13-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/pthread_h/3-3-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/pthread_h/8-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/pthread_h/13-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/pthread_h/2-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/pthread_h/3-8-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/pthread_h/14-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/pthread_h/10-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/pthread_h/20-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/pthread_h/4-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/pthread_h/5-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/pthread_h/7-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/pthread_h/3-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/pthread_h/1-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/pthread_h/3-9-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/pthread_h/3-4-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/pthread_h/6-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/pthread_h/3-5-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/pthread_h/3-10-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/pthread_h/12-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/pthread_h/3-6-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/pthread_h/18-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/pthread_h/3-7-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/pthread_h/9-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/interfaces/pthread_once/4-1-buildonly.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/stress/threads/sem_open/s-c1.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/stress/threads/sem_init/s-c1.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/stress/threads/pthread_cancel/stress.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/stress/threads/pthread_mutex_init/stress.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/stress/threads/pthread_mutex_init/s-c.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/stress/threads/pthread_mutex_lock/s-c1.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/stress/threads/pthread_mutex_lock/stress.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/stress/threads/pthread_mutex_lock/s-c2.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/stress/threads/pthread_once/stress.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/stress/threads/pthread_getschedparam/stress.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/stress/threads/sem_getvalue/stress.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/stress/threads/pthread_mutex_trylock/stress.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/stress/mqueues/multi_send_rev_2.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/stress/mqueues/multi_send_rev_1.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/stress/signals/sigismember_stress_1.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/stress/semaphores/multi_con_pro.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/mqueue_h/1-1.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/signal_h/26-1.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/sched_h/10-1.home.bashton.nuttx.wrk.apps.testing.ltp.o ltp/testcases/open_posix_testsuite/conformance/definitions/errno_h/4-1.home.bashton.nuttx.wrk.apps.testing.ltp.o
```